psalm-baseline.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.7.3@38c452ae584467e939d55377aaf83b5a26f19dd1">
+  <file src="src/Controller/Api/GalleryController.php">
+    <MissingDependency occurrences="3">
+      <code>Rest\View</code>
+      <code>Rest\View</code>
+      <code>Rest\View|FormInterface</code>
+    </MissingDependency>
+  </file>
+  <file src="src/Controller/Api/MediaController.php">
+    <MissingDependency occurrences="2">
+      <code>Rest\View</code>
+      <code>Rest\View|FormInterface</code>
+    </MissingDependency>
+  </file>
+  <file src="src/Listener/ODM/MediaEventSubscriber.php">
```

Comment:
For this other error, should I install doctrien/mongodb-odm?

Reply:
Why do we have `doctrine/mongodb-odm` if we're supposed to use `sonata-project/doctrine-orm-admin-bundle` with this bundle ?

Anyway, if it fix the error, I would add the dev dependency yes. It would check our conflict is the right one.

Reply:
This bundle integrates with phpcr, mongodb, and orm, but we are only adding the sonata-project orm bundle as require-dev. (And it is still missing doctrine/orm)

Reply:
Solved